sagemaker-featurestore/feature_store_introduction.ipynb

```diff
@@ -473,6 +473,152 @@
     "all_records"
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.            print("Waiting for Feature Group to be updated")
</pre><p>Branding nit - should it be "feature group", "FeatureGroup", or "Feature Group"? Use one consistently.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3515/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='46'/>

Reply:
<p>thx! fixed in new version.</p>